packages/node/.eslintrc.cjs

```diff
@@ -23,6 +23,19 @@ module.exports = {
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
 
     'no-console': 'error',
+
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      {
+        args: 'all',
+        argsIgnorePattern: '^_',
+        caughtErrors: 'all',
+        caughtErrorsIgnorePattern: '^_',
+        destructuredArrayIgnorePattern: '^_',
+        varsIgnorePattern: '^_',
+        ignoreRestSiblings: true,
+      },
+    ],
```

Comment:
couldn't spot any var names prefixed w underscores in our .ts files
so presuming this is for future use (e.g. try {} catch (_error) {} )

Reply:
stuff like that, but maybe not _that_ particular case, since you can just `try {} catch {}` now